tictactoe.js

```diff
@@ -0,0 +1,136 @@
+
+player= "X";
+space = document.getElementsByClassName("space");
+
+
+// var boardCheck;
+
+// var cellValue = function cells(key) {
+//     switch(key) {
+//         case 'a': return "";
+//         case 'b': return "";
+//         case 'c': return "";
+//         case 'd': return "";
+//         case 'e': return "";
+//         case 'f': return "";
+//         case 'g': return "";
+//         case 'h': return "";
+//         case 'i': return "";
+//         default : return "";
+//     }
+// }
+
+
+//-------------------------------------------------------------------
+function clickSpace() {
+    if (this.id == "a") {
+        if(document.getElementById("a").innerHTML == ""){ 
+           document.getElementById("a").innerHTML == player;
+           changeTurn();
+         }
+    } else if (this.id == "b") {
+        if(document.getElementById("b").innerHTML == ""){ 
+           document.getElementById("b").innerHTML = player;
+           changeTurn();
+         }
+    } else if (this.id == "c") {
+        if(document.getElementById("c").innerHTML == ""){ 
+           document.getElementById("c").innerHTML = player;
+           changeTurn();
+         }
+
+    } else if (this.id == "d") {
+      if(document.getElementById("d").innerHTML == ""){ 
+           document.getElementById("d").innerHTML = player;
+           changeTurn();
+         }
+    } else if (this.id == "e") {
+        if(document.getElementById("e").innerHTML == ""){ 
+           document.getElementById("e").innerHTML = player;
+           changeTurn();
+         }
+    } else if (this.id == "f") {
+        if(document.getElementById("f").innerHTML == ""){ 
+           document.getElementById("f").innerHTML = player;
+           changeTurn();
+         }
+
+    } else if (this.id == "g") {
+        if(document.getElementById("g").innerHTML == ""){ 
+           document.getElementById("g").innerHTML = player;
+           changeTurn();
+           }
+    } else if (this.id == "h") {
+        if(document.getElementById("h").innerHTML == ""){ 
+           document.getElementById("h").innerHTML = player;
+           changeTurn();
+         }
+    } else if (this.id == "i") {
+        if(document.getElementById("i").innerHTML == ""){ 
+           document.getElementById("i").innerHTML = player;
+           changeTurn();
+    }
+}
+    
+}
+
+document.getElementById("a").onclick = clickSpace;
+document.getElementById("b").onclick = clickSpace;
+document.getElementById("c").onclick = clickSpace;
+document.getElementById("d").onclick = clickSpace;
+document.getElementById("e").onclick = clickSpace;
+document.getElementById("f").onclick = clickSpace;
+document.getElementById("g").onclick = clickSpace;
+document.getElementById("h").onclick = clickSpace;
+document.getElementById("i").onclick = clickSpace;
+
+function changeTurn(){
+  if (player == 'x') {
+    player = 'o';
+  }  else {
+    player = 'x';
+  }
+  }
+
+
+function getWinner() {
+    if (isWinner('x')) {
```

Comment:
A return statement won't alert the players who won, so you could try adding an alert like this:
```
function getWinner() {
    if (isWinner('x')) {
      alert( 'x wins');
    }
    else if (isWinner('o')) {
      alert( 'o wins');
    }
}
```